applications/StructuralMechanicsApplication/custom_utilities/constitutive_law_utilities.cpp

```diff
@@ -316,6 +316,30 @@ double ConstitutiveLawUtilities<TVoigtSize>::CalculateCharacteristicLength(const
 /***********************************************************************************/
 /***********************************************************************************/
 
+template<SizeType TVoigtSize>
+double ConstitutiveLawUtilities<TVoigtSize>::CalculateCharacteristicLengthOnReferenceConfiguration(const GeometryType& rGeometry)
+{
+    double radius = 0.0;
+
+    const SizeType points_number = rGeometry.size();
+	array_1d<double, 3> center = rGeometry[0].GetInitialPosition().Coordinates();
+    for ( IndexType i_node = 1 ; i_node < points_number ; ++i_node ) {
+        center += rGeometry[i_node].GetInitialPosition().Coordinates();
```

Comment:
why don't you use the `Center`-Method of the geometry?

Reply:
is it because you use the ref-configuration?

Reply:
Yes, it is